<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLimit</class>
 <widget class="QDialog" name="UserLimit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户添加及权限设置</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameTxt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>权限设置：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>System</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Customer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VIPdesigner</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BusinessMen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>visitor</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="addUserBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userPwdTxt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="yesBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
